components/security/header/Header.jsx

```diff
@@ -1,15 +1,48 @@
 import React from "react";
 import Image from "next/image";
 import Link from "next/link";
-import { useUser } from "@auth0/nextjs-auth0";
 import Logop from "../../../public/icons/LogoPlataforma1.png";
 import styles from "../../../styles/Header.module.scss";
 import position from "../../../styles/Profile.module.scss";
 import styles2 from "../../../styles/Userdata.module.scss";
 
+let auth0 = {
+	domain: 'https://gethired.us.auth0.com',
+	response_type: 'token',
+	client_id: '9IKIwhKs44cPB0VceKTaVHZnXRSd2zUJ',
+	redirect_uri: ''
+}
+
 const Header = () => {
-	const { user } = useUser();
 
+	let [user, setUser] = React.useState(null);
+	React.useEffect(async () => {
```

Comment:
We might want to set the user data as context, for everybody to be able to consume